benches/http.rs

```diff
@@ -4,11 +4,11 @@ extern crate cookie_factory;
 
 use std::iter::repeat;
 
-use cookie_factory::*;
+
```

Comment:
nuke this empty line?

Reply:
Done. No idea why `cargo fix` didn't remove the empty line :)